### PR TITLE
Replaced "err" with  "nil" & added a comment

### PR DESCRIPTION
**Pull request by chawlanikhil24**
Signed-off-by: Nikhil Chawla <chawlanikhil24@gmail.com>

**- What I did**
fixed a typographical mistake, which was creating conflicts of thoughts while reading and understanding the code.

**- How I did it**
I read the code, and felt the need to replace it with something more appropriate. Even took the help from another docker contributor.

**- How to verify it**
The error can be raised in docker/cli/cobra.go
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**
rabbit

### CODE REVIEW

**Comment by chawlanikhil24** [comment]
@thaJeztah , Please review it. I guess this time its fine. :)

File: cli/cobra.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func SetupRootCommand(rootCmd *cobra.Command) {
 // docker/docker/cli error messages
 func FlagErrorFunc(cmd *cobra.Command, err error) error {
 	if err == nil {
-		return err
+		return nil //returning "nil" to ensure no error, and hence no need to continue
```
Comment (thaJeztah) [inline cli/cobra.go]: not sure the comment is really needed, but guess it's ok

**Review by thaJeztah — APPROVED** [review]
LGTM, but happy to hear from other maintainers if we should keep the comment.

Thanks!

**Comment by AkihiroSuda** [comment]
+1 for removing the comment

**Comment by ripcurld0** [comment]
+1 to @AkihiroSuda -- there is no added value to the comment.

**Comment by chawlanikhil24** [comment]
I have removed the comment as per the review. I Hope it can be merged now. @thaJeztah

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
can you squash your commits, so that there's a single commit in this PR? you can find instructions on doing so in our contributors guide; https://docs.docker.com/opensource/workflow/work-issue/#pull-and-rebase-frequently

**Comment by chawlanikhil24** [comment]
@thaJeztah , As instructed, squashed the commit.

**Review by thaJeztah — APPROVED** [review]
perfect! LGTM, thanks!

**Pull request merged**